src/style.css

```diff
@@ -1,3 +1,81 @@
 body {
+  background-color: #ddd;
+  color: black;
   margin: 0;
+  min-width: 1024px;
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+  --blue-color-for-link: #00acdc;
+}
+
+header {
```

Comment:
use classes instead of tags

Reply:
Sorry, but there is noted in checklist:
[CODE STYLE] - Don't use simple tag names or specific styles in class names. **Exception - specific semantic tags, like header, nav, footer etc.** Try to describe the content of the tag.

Reply:
you can use class `header` for sure, read again carefully:
  'Exception - specific semantic tags, like `header`, nav, footer etc.'

Reply:
Understood. Than you!